### PR TITLE
Load testing scripts

### PR DESCRIPTION
**Pull request by sastels**
# Description

Add scripts to help load test the app. Right now we have rate limiting turned on, so we can't go too crazy on it.

Requires k6 be installed on the computer via `brew install k6`
Usage:
```
LOAD_TESTING_BASE_URL=https://report-a-scam.cds-snc.ca k6 run -vu 10 -d10s loadTestingRequest.js
LOAD_TESTING_BASE_URL=https://report-a-scam.cds-snc.ca k6 run -vu 10 -d10s loadTestingSubmitNoFile.js
LOAD_TESTING_BASE_URL=https://report-a-scam.cds-snc.ca k6 run -vu 5 -d120s loadTestingSubmitFile.js
```

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code
- [x] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/utils/submitReportToServer.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
+var fs = require('fs')
 const flatten = require('flat')
 const fetch = require('isomorphic-fetch')
 const FormData = require('form-data')
 const { formDefaults } = require('../forms/defaultValues')
 
 async function submitReportToServer(url = '', data = {}) {
   const flattenedData = flatten(data, { safe: true })
+  console.log(flattenedData)
```
Comment (sastels) [inline f2/src/utils/submitReportToServer.js:9]: useful for getting the data fields for the k6 scripts. k6 didn't want to include `formDefaults` in the script unfortunately. Nor `flatten`, for that matter. 😞

File: f2/src/forms/defaultValues.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const formDefaults = {
   prodVersion: '',
   appVersion: '',
   consent: { consentOptions: [] },
-  anonymous: { anonymousOptions: [] },
+  anonymous: { anonymousOptions: ['anonymousPage.no'] },
```
Comment (sastels) [inline f2/src/forms/defaultValues.js:6]: the report email formatter fails if `anonymousOptions` is empty.

File: utils/loadTestingSubmitNoFile.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  "whatHappened.whatHappened": "",
+  "suspectClues.suspectClues1": "",
+  "suspectClues.suspectClues2": "",
+  "suspectClues.suspectClues3": "",
+  "evidence.files": [],
+  "evidence.fileDescriptions": [],
+  "location.postalCode": "",
+  "location.city": "",
+  "location.province": "",
+  "contactInfo.fullName": "Load Test",
```
Comment (DianeLiu2019) [inline utils/loadTestingSubmitNoFile.js:52]: I just curious why need add "Load Test" here instead of leave it empty. I changed to empty "" and run the load test again. it shows  check is 94.91% instead of 100%
